Fix vcenter issues

naumvd95 opened the pull request:
https://mirantis.jira.com/browse/FPL-2710
desc:
1) bad substitution with ":?" bash wrap
2) seems bad defining variables in vcenter reporter
3)fix wrong test_suite naming
see desc and comments in JIRA dashboard

naumvd95 committed:
https://mirantis.jira.com/browse/FPL-2710

desc:
1) bad substitution with ":?" bash wrap
2) seems bad defining variables in vcenter reporter
3)fix wrong test_suite naming
see desc and comments in JIRA dashboard

rkhozinov commented on servers/partner-ci/9.x/vcenter.yaml:
```diff
@@ -81,14 +81,14 @@
     testrail_suite_desc: "Suite description"
     testrail_milestone: "9.2"
-    testrail_plan: "$TREP_TESTRAIL_MILESTONE snapshot #${{SNAPSHOTS_ID}}"
+    testrail_plan: "${{TREP_TESTRAIL_MILESTONE}} snapshot #${{SNAPSHOTS_ID}}"
```
SNAPSHOTS_ID and TREP_TESTRAIL_MILESTONE are parameters and will be used as parameters.
Jenkins job parse them as bash variables before job run. 
Do you check it on jenkins?

rkhozinov replied:
it's ok

rkhozinov commented on servers/partner-ci/9.x/vcenter.yaml:
```diff
@@ -166,7 +166,18 @@
           testgroup_value: '{obj:testgroup}'
 
-      - '{reporter-name}'
+      - '{reporter-name}':
```
Reporter should use global variables, we don't need duplicate pieces of the code

naumvd95 replied:
i know , its just for testing

rkhozinov replied:
ok

rkhozinov commented on servers/partner-ci/builders/vcenter.test.sh:
```diff
@@ -62,7 +62,7 @@ if [[ $ISO_FILE == *"Mirantis"* ]]; then
   export FUEL_RELEASE=$(echo $ISO_FILE | cut -d- -f2 | tr -d '.iso')
 fi
 
-export ENV_NAME="${ENV_PREFIX}.${SNAPSHOTS_ID:?}"
```
what's the problem?

rkhozinov replied:
attach a trace

naumvd95 replied:
repeat
SEE FULL DESCRIPTION IN JIRA tASK

rkhozinov replied:
`line 65: SNAPSHOTS_ID: parameter null or not set ` 
`export SNAPSHOTS_ID= ` - it's the problem
```
+ [[ 610 == *\l\a\s\t\S\u\c\c\e\s\s\f\u\l\B\u\i\l\d* ]]
+ export SNAPSHOTS_ID= << it's the problem
+ SNAPSHOTS_ID=
+ '[' -f build.plugin_version ']'
++ grep PLUGIN_VERSION
++ cut -d= -f2
+ version=3.2.0
+ export DVS_PLUGIN_VERSION=3.2.0
+ DVS_PLUGIN_VERSION=3.2.0
+ '[' -z 116 ']'
+ '[' -f nosetest.xml ']'
+ sudo rm -rf 'logs/*'
+ [[ MirantisOpenStack-9.0.iso == *\M\i\r\a\n\t\i\s* ]]
++ echo MirantisOpenStack-9.0.iso
++ tr -d .iso
++ cut -d- -f2
+ export FUEL_RELEASE=90
+ FUEL_RELEASE=90
/tmp/hudson722437583800698906.sh: line 65: SNAPSHOTS_ID: parameter null or not set
```

rkhozinov commented on servers/partner-ci/common.yaml:
```diff
@@ -875,8 +875,8 @@
           mask-password-params: true
     parameters:
       - string:
-          name: ISO_VERSION
```
Are you sure that it won't affect previous releases?

naumvd95 replied:
sorry its mistake

naumvd95 committed:
common: fix mistake with removing param

Commit:
vcenter.test: some issues are fixed

- we don't have custom iso anymore
- snapshots_id export logic is fixed
- some style fixes

naumvd95 commented on servers/partner-ci/builders/vcenter.test.sh:
```diff
@@ -28,8 +24,7 @@ if [ "${SNAPSHOTS_ID}" != "released" ]; then
 [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]] && \
-  id=$(grep -Po '#\K[^ ]+' < snapshots.params)
-  export SNAPSHOTS_ID=$id
+  export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params)
```
ok

naumvd95 commented on servers/partner-ci/builders/vcenter.test.sh:
```diff
@@ -124,8 +119,8 @@ setup_bridge() {
   if sudo iptables-save | grep $bridge | grep -i reject | grep -q FORWARD; then
-    iptables -D FORWARD -o $bridge -j REJECT --reject-with icmp-port-unreachable
-    iptables -D FORWARD -i $bridge -j REJECT --reject-with icmp-port-unreachable
+    sudo iptables -D FORWARD -o $bridge -j REJECT --reject-with icmp-port-unreachable
```
perfect

naumvd95 committed:
vcenter: reporter fix

- reporter variables fix
- brackets refactoring

Commit:
vcenter.test: style fixes, refactoring
